.github/workflows/update_dependency_version.yml

```diff
@@ -41,10 +41,7 @@ jobs:
       -   name: "[Scheduled] Update Ballerina Language Version"
           if: github.event_name == 'schedule'
           run: |
-            python dependabot/update_dependencies_version.py "${{ github.event.inputs.isRetrigger }}" \
-                                                              "${{ github.event.inputs.ballerinaVersion }}" \
-                                                              "${{ github.event.inputs.autoMergePRs }}" \
-                                                              "schedule"
+            python dependabot/update_dependencies_version.py "false" "" "true" "schedule"
```

Comment:
There are 3 variables passed here and 4 passed to the other?

Reply:
$th argument is optional